package.json

```diff
@@ -84,7 +84,7 @@
     "falcor-asap": "2.0.6",
     "falcor-json-graph": "1.1.7",
     "falcor-path-syntax": "0.2.4",
-    "falcor-path-utils": "0.5.1",
+    "falcor-path-utils": "0.7.2",
```

Comment:
Updated to pull in a [bugfix in iterateKeySet](https://github.com/Netflix/falcor-path-utils/commit/5e9d7e01849e9ab6b757627b8c701627e9d6b825).